packages/cli/templates/js/src/main.ts

```diff
@@ -0,0 +1,206 @@
+import {
```

Comment:
What do you think? Let me know if it's too much of abstraction. I can make it more plain.

Reply:
See the comment on this whole file :)